assets/stories/skate-components/routePill.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import type { Meta, StoryObj } from "@storybook/react"
+
+import { RoutePill } from "../../src/components/routePill"
+
+const meta = {
+  component: RoutePill,
+} satisfies Meta<typeof RoutePill>
+
+export default meta
+type Story = StoryObj<typeof RoutePill>
+
+export const Default: Story = {
+  args: { routeName: "66" },
+}
```

Comment:
I'd be interested in a few additional stories that show off some of the ways that the route pill can be styled, like 👇 possibly

```suggestion
export const Default: Story = {
  args: { routeName: "66" },
}

export const SilverLine: Story = {
  args: { routeName: "SL3" },
}

export const BlueLine: Story = {
  args: { routeName: "Blue Line" },
}

export const OrangeLine: Story = {
  args: { routeName: "Orange Line" },
}

export const RedLine: Story = {
  args: { routeName: "Red Line" },
}

export const GreenLine: Story = {
  args: { routeName: "D" },
}
```

I'm on the fence though, because we're still under a snapshot limit with Chromatic, which means that adding a bunch of stories will stress that even more.

Reply:
🤷 unsure! I'd need more details to decide what to do here -- how far are we from the snapshot limit? I didn't think of adding all these stories, given the sizing changes were the focus of the task, but I'd be ok with rolling this in to chip away at tech debt.

Reply:
Eh I'm good either way - I do see your point that adding these is outside the scope of this ticket, so I'm game with punting it.

I'd also be fine with adding just one of those, in order to show off the fact that the route pill changes color when given route names that are not normal bus routes.

Reply:
I like that as a compromise! Low risk to add just one more story without overthinking!

```suggestion
export const Default: Story = {
  args: { routeName: "66" },
}

export const BlueLine: Story = {
  args: { routeName: "Blue Line" },
}
```